modules/cloud-config-container/simple-nva/files/frr/frr.conf

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default FRR configuration which enables a BGP protocol process with a default ASN of 65500.
+# For more information on how to update this file and/or configure bgp sessions please refer to 
+# the official documentation available at the following link:
+# https://docs.frrouting.org/en/latest/overview.html
+
+log syslog informational
+no ipv6 forwarding
+router bgp 65500
```

Comment:
I'm wondering if we should make this a variable to be used if bgp is enabled. It's likely the most common thing hat users will need to change

Reply:
I would not introduce a variable just for updating the ASN since it will be used only in the sample deployment. Every time you use this module  you would provide a custom frr.conf file with custom configuration, isn't it?

Reply:
ah ok, sg. If that's the idea we should make it clear in the readme.

Reply:
makes sense. Plus, as per our discussion (also reported at the comment above), we'll remove this and we'll make the frr_config mandatory in case someone decides to enable advanced routing (frr).

Reply:
removed default frr.conf file and place an example in README